File: Ent_UsAdmin/agregarservicio.vb
Public Class agregarservicio

   






    Private Sub agregarservicio_FormClosed(sender As Object, e As FormClosedEventArgs) Handles Me.FormClosed
        inv.catservicios.MonoFlat_Label1.ForeColor = Color.FromArgb(64, 64, 64)
        inv.catservicios.MonoFlat_Label1.BorderStyle = BorderStyle.None
        inv.catservicios.MonoFlat_Label1.Location = New Point(inv.catservicios.MonoFlat_Label1.Location.X, inv.catservicios.yiniciallabel)
        inv.catservicios.bloquearlabelagregar = False
        inv.catservicios.agregarservicioabierto = False
    End Sub

    Private Sub Panel1_MouseDown(sender As Object, e As MouseEventArgs) Handles Panel1.MouseDown
        If e.Button = Windows.Forms.MouseButtons.Left Then
            MoveForm(Me)
        End If
    End Sub


   
    

    Private Sub Panel1_Paint(sender As Object, e As PaintEventArgs) Handles Panel1.Paint

    End Sub

    Private Sub agregarservicio_Load(sender As Object, e As EventArgs) Handles MyBase.Load

    End Sub
End Class

File: Ent_UsAdmin/Catalogos/agregarservicio.cs
using System;
using System.Drawing;
using System.Windows.Forms;

namespace ConfiaAdmin
{
    public partial class agregarservicio
    {
        public agregarservicio()
        {
            InitializeComponent();
        }








        private void agregarservicio_FormClosed(object sender, FormClosedEventArgs e)
        {
            My.MyProject.Forms.inv.catservicios.MonoFlat_Label1.ForeColor = Color.FromArgb(64, 64, 64);
            My.MyProject.Forms.inv.catservicios.MonoFlat_Label1.BorderStyle = BorderStyle.None;
            My.MyProject.Forms.inv.catservicios.MonoFlat_Label1.Location = new Point(My.MyProject.Forms.inv.catservicios.MonoFlat_Label1.Location.X, My.MyProject.Forms.inv.catservicios.yiniciallabel);
            My.MyProject.Forms.inv.catservicios.bloquearlabelagregar = false;
            My.MyProject.Forms.inv.catservicios.agregarservicioabierto = false;
        }

        private void Panel1_MouseDown(object sender, MouseEventArgs e)
        {
            if (e.Button == MouseButtons.Left)
            {
                Module_resize.MoveForm(this);
            }
        }





        private void Panel1_Paint(object sender, PaintEventArgs e)
        {

        }

        private void agregarservicio_Load(object sender, EventArgs e)
        {

        }
    }
}